cypress.config.js

```diff
@@ -0,0 +1,28 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  video: true,
+  defaultCommandTimeout: 150000,
+  env: {
+    API_BASE_URL: 'http://localhost/openmrs/ws/rest/v1',
+    ADMIN_USERNAME: 'admin',
+    ADMIN_PASSWORD: 'Admin123',
```

Comment:
While it's somewhat irrelevant here (our default user name and password are well known), the best practice would be to actually feed these into the environment at run time via GitHub secrets.

Reply:
+1 for the suggestion. We need to add this to a backlog.